docs/registry/create/_index.md

```diff
@@ -11,154 +11,110 @@ tags:
     "modular resources",
     "components",
     "services",
+    "registry",
   ]
-description: "Use the Viam module system to implement modular resources that can be included in any Viam-powered machine."
+description: "Create a module to provide a new modular resource to your Viam machine."
 aliases:
   - "/extend/modular-resources/create/"
   - "/modular-resources/create/"
 no_list: true
 ---
 
-Viam provides built-in support for a variety of different {{< glossary_tooltip term_id="component" text="components" >}} and {{< glossary_tooltip term_id="service" text="services" >}}, but you can also add support for unsupported resources by creating a module.
-A _module_ provides one or more {{< glossary_tooltip term_id="modular-resource" text="modular resources" >}}, which add support for {{< glossary_tooltip term_id="resource" text="resource" >}} {{< glossary_tooltip term_id="type" text="types" >}} or {{< glossary_tooltip term_id="model" text="models" >}} that are not built into Viam.
+Viam provides built-in support for a variety of different {{< glossary_tooltip term_id="component" text="components" >}} and {{< glossary_tooltip term_id="service" text="services" >}} -- collectively called {{< glossary_tooltip term_id="resource" text="resources" >}} -- but you might want to use Viam with a new component type or service model that isn't currently supported.
+To extend support to these resources, you can create a {{< glossary_tooltip term_id="module" text="module" >}}.
```

Comment:
I don't know what this sentence is supposed to mean. Maybe?
```suggestion
If the existing resources do not support your hardware or service requirements, you can create a {{< glossary_tooltip term_id="module" text="module" >}}.
```

Reply:
Trying something different, LMK what you think.